integrations/apos_importer.py

```diff
@@ -265,35 +264,102 @@ def create_facetter_and_klasser(self):
 
         if CREATE_UDVALGS_CLASSES:
             specific_klasser = [
-                {'titel': 'AMR', 'facet': 'org_unit_type', 'scope': 'TEXT'},
-                {'titel': 'H-MED', 'facet': 'org_unit_type', 'scope': 'TEXT'},
-                {'titel': 'C-MED', 'facet': 'org_unit_type', 'scope': 'TEXT'},
-                {'titel': 'L-MED', 'facet': 'org_unit_type', 'scope': 'TEXT'}
+                {'id': 'AMR', 'titel': 'AMR',
+                 'facet': 'org_unit_type', 'scope': 'TEXT'},
+                {'id': 'H-MED', 'titel': 'H-MED',
+                 'facet': 'org_unit_type', 'scope': 'TEXT'},
+                {'id': 'C-MED', 'titel': 'C-MED',
+                 'facet': 'org_unit_type', 'scope': 'TEXT'},
+                {'id': 'L-MED', 'titel': 'L-MED',
+                 'facet': 'org_unit_type', 'scope': 'TEXT'}
             ]
         else:
             specific_klasser = []
         standard_klasser = [
```

Comment:
standard-klasser? 
Jo - når man importerer fra Apos får man apos klassifikationssystem med, men ikke disse, og dem skal man bruge til diverse udvalg. Udvalg håndteres for nærværende udenfor Apos, derfor denne ekstra klass-oprettelse. 

Jeg fik det at vide, fordi jeg undrede mig over at Leder var udkommenteret - den kommer nemlig direkte fra Apos.